app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterMultipleClaimsController.scala

```diff
@@ -78,104 +77,106 @@ class EnterMultipleClaimsController @Inject() (
 
   import EnterMultipleClaimsController._
 
-  def enterClaim(mrnIndex: Int, taxCode: TaxCode): Action[AnyContent] =
+  def enterClaim(implicit journey: JourneyBindable, mrnIndex: Int, taxCode: TaxCode): Action[AnyContent] =
```

Comment:
I'll move the implicit to the end like this: 
```
  def enterClaim(mrnIndex: Int, taxCode: TaxCode)(implicit journey: JourneyBindable): Action[AnyContent] =
```

Then you won't have to pass `journey` explicitly like you do here: 
```.enterClaimSubmit(journey, mrnIndex, claim.taxCode)```

You'll have to make this change in a few other places.

Reply:
Ok - will make the changes